setup.py

```diff
@@ -11,14 +11,19 @@
     author_email='johan.dahlberg@medsci.uu.se',
     url="https://www.github.com/Molmed/checkQC",
     download_url='https://github.com/Molmed/checkQC/archive/{}.tar.gz'.format(__version__),
-    python_requires='>3.10, <3.11',
```

Comment:
It would be nice to update the GHA workflow to check if tests works for 3.11, 3.12 and 3.13 as well :) 
This is not necessary though